### PR TITLE
[docs] Update UI libraries list

### PR DESCRIPTION
**Pull request by gabimoncha**
# Why

UI lib list needs update

# How

Create another table with unmaintained libraries and add Ignite UI and React Native Reusables

# Test Plan

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by Simek — COMMENTED** [review]
Hello @gabimoncha, thanks for the contribution! 👍 

Since [UI Kitten](https://github.com/akveo/react-native-ui-kitten) latest release was mid last year, I would not consider this library as an unmaintained yet. Let's move it back to main table, and drop iOS Kit with the added "Unmaintained" table all together. 

The rest of the changes look great.

**Comment by gabimoncha** [comment]
@Simek done!

File: docs/pages/ui-programming/user-interface-libraries.mdx [hunk 1]
```diff
@@ -20,12 +20,6 @@ Below are some of the most popular UI Libraries. Test them all out and see what
 | [Gluestack UI](https://ui.gluestack.io/)                                          | [Link](https://ui.gluestack.io/)                                                                                |
 | [Ignite UI](https://docs.infinite.red/ignite-cli/)                                | [Link](https://docs.infinite.red/ignite-cli/boilerplate/components/)                            |
 | [React Native Reusables](https://github.com/mrzachnugent/react-native-reusables)  | [Link](https://github.com/mrzachnugent/react-native-reusables)                            |
+| [React Native UI Kitten](https://github.com/akveo/react-native-ui-kitten)         | [Link](https://akveo.github.io/react-native-ui-kitten/docs/getting-started/what-is-ui-kitten#what-is-ui-kitten) |
 
 > As with any library, it's always best to find one with a large and active community. This way, help is easy to come by. Check out any project's GitHub page to see what the activity is like.
```
Comment (amandeepmittal) [inline docs/pages/ui-programming/user-interface-libraries.mdx:25]:
```suggestion
When selecting a library, select one that is supported by a large and active community. Check out a library's GitHub page to see its recent activity.
```

- This doesn't have to be a callout
- Let's be concise here in the message about choosing a library

Comment (Simek) [reply docs/pages/ui-programming/user-interface-libraries.mdx:25]: This note was not edited/added in this changeset, but let's merge it in. 🙂

**Review by Simek — APPROVED** [review]
Thanks for the updates @gabimoncha! Let's ship this. 🚀

File: docs/pages/ui-programming/user-interface-libraries.mdx (hunk 1 above)
Comment (amandeepmittal) [reply docs/pages/ui-programming/user-interface-libraries.mdx]: Oh my bad! Thanks for catching that!

**Pull request merged**